keyboards/40percentclub/gherkin/keymaps/midi/config.h

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright 2012 Jun Wako <wakojun@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef CONFIG_USER_H
```

Comment:
A candidate for `#prama once`

Reply:
I'm not sure I follow what this does (again, sad to say this is the first C I've written in forever), but I'm very happy to implement?

Reply:
Both header guards and `#pragma once` are trying to solve the same thing, including the files content only once.

```c
#ifndef CONFIG_USER_H
#define CONFIG_USER_H

// some header content...

#endif
```

vs

```c
#pragma once

// some header content...
```

The latter is the QMK preferred style. Personally I find it less error prone and friendlier if you rename/move the file, and most importantly, less to type.

Reply:
Ah! That makes it very clear, thanks!